requests/script.js

```diff
@@ -0,0 +1,505 @@
+    reason,
+    updatedAt,
+  } = oooRequest;
+  let showAdminDetailsClass = 'notHidden';
+  let showActionButtonClass = 'notHidden';
+  if (
+    state === 'PENDING' ||
+    lastModifiedBy === undefined ||
+    lastModifiedBy === null
+  ) {
+    showAdminDetailsClass = 'hidden';
+  }
+  if (state !== 'PENDING') {
+    showActionButtonClass = 'hidden';
+  }
+  const createdDate = convertDateToReadableStringDate(
+    createdAt,
+    DEFAULT_DATE_FORMAT,
+  );
+  const createdDateInAgoFormat = dateDiff(
+    Date.now(),
+    createdAt,
+    (s) => s + ' ago',
+  );
+  const fromDate = convertDateToReadableStringDate(from, DEFAULT_DATE_FORMAT);
+  const toDate = convertDateToReadableStringDate(until, DEFAULT_DATE_FORMAT);
+  let updatedDate = convertDateToReadableStringDate(
+    updatedAt,
+    DEFAULT_DATE_FORMAT,
+  );
+  const updatedDateInAgoFormat = dateDiff(
+    Date.now(),
+    updatedAt,
+    (s) => s + ' ago',
+  );
+
+  const card = createElementFromMap({
+    tagName: 'div',
+    class: 'ooo_request__card',
+    child: [
+      generateRequesterInfo(),
+      generateRequestContent(),
+      addHotizontalBreakLine(),
+      generateSuperuserInfo(),
+      generateActionButtonsContainer(),
+    ],
+  });
+  return card;
+
+  function addHotizontalBreakLine() {
+    return createElementFromMap({
+      tagName: 'hr',
+      class: 'horizontal__line__saperator',
+    });
+  }
+
+  function generateActionButtonsContainer() {
+    return createElementFromMap({
+      tagName: 'div',
+      class: ['action__container', showActionButtonClass],
+      child: [
+        createElementFromMap({
+          tagName: 'input',
+          class: 'request__remark__input',
+          id: `remark-text-${id}`,
+          type: 'text',
+          placeholder: 'Add Remark If Any...',
+        }),
+        createElementFromMap({
+          tagName: 'div',
+          class: ['action__buttons__container'],
+          child: [
+            createElementFromMap({
+              tagName: 'button',
+              class: ['request__action__btn', 'accept__btn'],
+              id: `${id}`,
+              textContent: 'Accept',
+              eventListeners: [
+                {
+                  event: 'click',
+                  func: (e) => performAcceptRejectAction(true, e),
+                },
+              ],
+            }),
+            createElementFromMap({
+              tagName: 'button',
+              class: ['request__action__btn', 'reject__btn'],
+              id: `${id}`,
+              textContent: 'Reject',
+              eventListeners: [
+                {
+                  event: 'click',
+                  func: (e) => performAcceptRejectAction(false, e),
+                },
+              ],
+            }),
+          ],
+        }),
+      ],
+    });
+  }
+
+  function generateSuperuserInfo() {
+    return createElementFromMap({
+      tagName: 'div',
+      class: ['admin__info__and__status', showAdminDetailsClass],
```

Comment:
```suggestion
      class: ['admin__info__and__status', showSuperuserDetailsClass],
```

Reply:
The variable name is changed. please check.